grgit-gradle/src/main/groovy/org/ajoberstar/grgit/gradle/GrgitPlugin.groovy

```diff
@@ -22,15 +22,14 @@ class GrgitPlugin implements Plugin<Project> {
         grgit.close()
       }
 
-      project.allprojects { prj ->
-        if (prj.ext.has('grgit')) {
+      project.allprojects { Project prj ->
+        if (prj.extensions.hasProperty('grgit')) {
           prj.logger.warn("Project ${prj.path} already has a grgit property. Remove org.ajoberstar.grgit from either ${prj.path} or ${project.path}.")
         }
-        prj.ext.grgit = grgit
+        prj.extensions.add(Grgit,"grgit", grgit)
```

Comment:
Nit: Could / should use single quotes for the non-interpolated string here.

Reply:
Is there some sort of performance improvement using one vs the other?

Reply:
I would guess so, as single-quoted strings to not need to be parsed for variables to expand, but I have no numbers to underpin that, and for sure it is a micro-optimization, but nevertheless good practice, IMO.

Reply:
Just for the record: Single quotes is the preferred style but only microscopic difference performance wise. There are a few extra cycles needed early on during parsing where no placeholders will be found (in a case like this) and both become the same thing after that. The slight preference is for human readers knowing that don't need to look ahead for dollar placeholders.